web/react/utils/constants.jsx

```diff
@@ -479,6 +479,14 @@ export default {
         EMBED_PREVIEW: {
             label: 'embed_preview',
             description: 'Show preview snippet of links below message'
+        },
+        EMBED_TOGGLE: {
+            label: 'embed_toggle',
+            description: 'Show toggle for all embed previews'
+        },
+        LOC_PREVIEW: {
```

Comment:
Yeah I think this might have slipped through the cracks in my rebase. Should I just remove `LOC_PREVIEW` or the `EMBED_TOGGLE` too? I can't tell if the features are still still used from this file at all.